src/preview/webview.ts

```diff
@@ -9,6 +9,7 @@ import { getScriptContent } from "./languages/script";
 import { getCDNJSLibraries } from "./libraries/cdnjs";
 import { ProxyFileSystemProvider } from "./proxyFileSystemProvider";
 import { storage } from "./tutorials/storage";
+import vscodeStyles from './vscodeStyles';
```

Comment:
Maybe call this file `themeStyles`? That way it's a bit clearer that it's indicating the styles for the user's theme? I don't feel strongly either way though :)